src/gui/mzroll/loginform.cpp

```diff
@@ -31,9 +31,10 @@ WorkerThread::WorkerThread()
     
 };
 
-void WorkerThread::run(){
-    QString status_inside = _pollyintegration->authenticateLogin(username,password);
-    emit resultReady(QStringList()<<status_inside<<username<<password);
+void WorkerThread::run()
+{
+    QString status_inside = _pollyintegration->authenticateLogin(username, password);
+    emit resultReady(QStringList() << status_inside << username <<password);
```

Comment:
@Ky5595 Don't need to add username and password to this qstringlist either right? It can just say status_inside?